src/test/java/uk/gov/digital/ho/hocs/search/queue/SearchConsumerTest.java

```diff
@@ -142,6 +180,16 @@ public void shouldCallDeleteTopic() throws JsonProcessingException {
         verifyNoMoreInteractions(mockDataService);
     }
 
+    @Test
+    public void shouldCallDeleteTopic() throws JsonProcessingException {
+
+        CreateAuditDto auditDto = new CreateAuditDto(caseUUID, "\"11111111-2222-2222-2222-333333333333\"", EventType.CASE_TOPIC_DELETED.toString());
+        String json = mapper.writeValueAsString(auditDto);
+        template.sendBody(searchQueue, json);
+        verify(mockDataService, times(1)).deleteTopic(eq(caseUUID), eq("11111111-2222-2222-2222-333333333333"));
+        verifyNoMoreInteractions(mockDataService);
+    }*/
```

Comment:
Why are these commented out?

Reply:
Very good question, they are copied and pasted from above for me to work off. Can be removed as the test is still present and I didn't use it.

Reply:
This has been rectified 👍🏼